packages/pds/src/auth.ts

```diff
@@ -27,92 +33,118 @@ export type AuthToken = {
   scope: AuthScope
   sub: string
   exp: number
+  aud?: string
 }
 
-export type RefreshToken = AuthToken & { jti: string }
+export type RefreshToken = AuthToken & { jti: string; aud: string }
 
 export class ServerAuth {
-  private _secret: string
+  private _signingSecret: KeyObject
+  private _signingKeyPromise?: Promise<KeyObject>
```

Comment:
i'm inclined to put this behind an async static constructor so we don't have to manage the promise